src/oracle/query_dsl/mod.rs

```diff
@@ -1 +1,4 @@
+mod returning;
 mod save_changes_dsl;
+
+pub use self::returning::{OciReturningClause, OciReturningDsl};
```

Comment:
Add `impl SupportReturningClause for OciConnection` in `connection.rs`

Reply:
That does unfortunately not add support for the normal returning syntax in diesel itself, it just implements some functionality that constructs returning queries.
We can change this as soon as the re-rebalance rfc lands.

Reply:
Maybe we should have an issue for that.